python/test/test_typing.py

```diff
@@ -49,6 +49,7 @@ def test_sensitivity():
     assert isinstance(L1Distance[int], SensitivityMetric)
     assert not isinstance(RuntimeType.parse('(f32)'), SensitivityMetric)
     assert str(L1Distance[int]) == "L1Distance<i32>"
+    assert L1Distance[int] != {}
```

Comment:
The assert just above would seem to guarantee that this would also pass: Not sure what it adds?

Reply:
It makes it so that code coverage is complete- a RuntimeType should not generally be equal to non-RuntimeTypes.